### PR TITLE
Added coturn formula

### PR DESCRIPTION
**Pull request by bfanger**
A free open source implementation of TURN and STUN Server

Which is used for reliable (WebRTC) connections.

### CODE REVIEW

File: Library/Formula/coturn.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+class Coturn < Formula
+  desc "Free open source implementation of TURN and STUN Server"
+  homepage "https://github.com/coturn/coturn"
+  url "http://turnserver.open-sys.org/downloads/v#{version}/turnserver-#{version}.tar.gz"
```
Comment (DomT4) [inline Library/Formula/coturn.rb]: We don't do this in Homebrew. The `version` should be inlined into the URL.

**Comment by bfanger** [comment]
@DomT4 Ah, I misinterpreted the "version 4.4.5.4 is redundant with version scanned from URL" audit error.
inlined the version into the url and removed the version property.

File: Library/Formula/coturn.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--mandir=#{man}",
+                          "--prefix=#{prefix}"
+
+    system "make", "install"
+  end
+
+  test do
+    system "turnadmin", "-l"
```
Comment (DomT4) [inline Library/Formula/coturn.rb]: This needs to be `system bin/"turnadmin", "-l"`, presuming the binary is in `bin`. We use full paths in testing.

Comment (bfanger) [inline Library/Formula/coturn.rb]: Changed `system "turnadmin", "-l"` to `system "#{bin}/turnadmin", "-l"`

**Comment by bfanger** [comment]
:white_check_mark: Copy-pastable url
 :white_check_mark: Using absolute path in `test do`

Anything else?

**Comment by bfanger** [comment]
Travis-CI failed due to github rate-limiting, you might want to add a `HOMEBREW_GITHUB_API_TOKEN`
to your travis environment to prevent this in the future:

http://docs.travis-ci.com/user/environment-variables/#Encrypted-Variables

Restart the job when github has calmed down, and you'll see that the formula builds just fine.

**Comment by DomT4** [comment]
We're aware of the Travis issues and working on them. Jenkins remains the primary CI at the moment.

**Comment by dunn** [comment]
Thanks for your patience! Merged in 22cccc48c6803076e8eb607540d15625cb47b842.

**Pull request closed without merging**